src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -524,6 +534,17 @@ protected void ExecuteIndexing()
                             Log.ErrorException($"Could not update stats for '{Name} ({IndexId})'.", e);
                         }
 
+                        _indexingInProgress = false;
+
+                        try
+                        {
+                            MaybePersistQueryingTime();
```

Comment:
Why persist the last query time ? Except for tracking activity across startups, is there a reason?